views/term.erb

```diff
@@ -4,11 +4,14 @@
         <p><%= @term['start_date'] %> – <%= @term['end_date'] %></p>
 
       <% if @memberships %>
-        <ul>
+        <ul class="avatar-list">
           <% @memberships.sort_by { |m| m['person']['name'] }.each do |m| %>
             <li>
-                <a href="<%= person_url(m['person']) %>"><%= m['person']['name'] %></a>
-                (<%= m['start_date'] %>–<%= m['end_date'] %>)
+               <a href="<%= person_url(m['person']) %>">
+                 <span class="avatar"><i class="fa fa-user"></i></span>
+                 <span class="label"><%= m['person']['name'] %></span>
+                 (<%= m['start_date'] %>–<%= m['end_date'] %>)
+               </a>
```

Comment:
the link should really close _before_ the dates, rather than after, partially because it makes more sense to (as the dates might want to be links to something else), but also because it causes some styling confusion otherwise (the names get an underscore on hover, but the dates don't)